.github/workflows/ci.yml

```diff
@@ -33,14 +33,22 @@ jobs:
         -   uses: actions/checkout@v4
         -   name: "Main Script"
             run: |
+                set -x
                 CONDA_ENVIRONMENT=.test-conda-env-py3.yml
                 echo "- matplotlib" >> $CONDA_ENVIRONMENT
                 echo "- pyopengl" >> $CONDA_ENVIRONMENT
                 echo "- ipython" >> $CONDA_ENVIRONMENT
 
                 curl -L -O https://tiker.net/ci-support-v0
                 . ci-support-v0
-                build_py_project_in_conda_env
+
+                install_conda_deps
+
+                export CL_INC_DIR="$CONDA_PREFIX/include"
+                export CL_LIB_DIR="$CONDA_PREFIX/lib"
+                export CL_LIBNAME=OpenCL
```

Comment:
Could this logic be in `CMakeLists.txt`? Something like `if($ENV{CONDA_PREFIX}) set(CL_INC_DIR "${CONDA_PREFIX}/include")`?

Reply:
I agree, but maybe the logic could be made conditional on there being an OpenCL library in the conda env? If there is, I think there is an overwhelming likelihood it should be used. And PyOpenCL's documentation talks about conda as a means to install, so it's not like the two projects are totally unrelated.

Reply:
For context, on my system (cmake 3.29.3) there are a few cmake modules that look at `CONDA_PREFIX` in particular: `GNUInstallDirs.cmake`, `UnixPaths.cmake` and `FindPython/Support.cmake`.

So I don't think there's anything strange about teaching the build a bit more about the platform, which in this case is `conda`.

Reply:
Also, looking at `FindOpenCL.cmake`, it seems like we can set `OCL_ROOT=$CONDA_PREFIX` to let it find the right folders itself.

Reply:
TIL, thanks. conda support was added in [9b3c600](https://github.com/inducer/pyopencl/pull/755/commits/9b3c6006492b298882bcb0630389382e5262cd53)